### PR TITLE
replace spaces in urls

### PR DESCRIPTION
**Pull request by revirtux**
now there are no spaces in the URLs of houses and categories  #31

### CODE REVIEW

File: index/views.py [hunk 1]
```diff
@@ -69,8 +71,12 @@ def dynamic_css(request, house_name):
     resp['Content-Type'] = 'text/css'
     return resp
 
+
 def challenges_path(request, category_name):
-    category_list = categoriesmanager.get_all_categories().filter(name=category_name)
+    category_name = category_name.replace("-", " ")
+    print(category_name)
+    category_list = categoriesmanager.get_all_categories().filter(
+        name=category_name.replace("-", " "))
```
Comment (revirtux) [inline index/views.py]: oppsy

Comment (revirtux) [reply index/views.py]: fixed!

File: index/views.py [hunk 2]
```diff
@@ -44,6 +44,8 @@ def index(request):
 
 
 def house_path(request, house_name):
+    house_name = house_name.replace("-", " ")
```
Comment (AmitTPB) [inline index/views.py:47]: This ruins the css files somehow, fix it
image

**Review by AmitTPB — CHANGES_REQUESTED** [review]

File: index/views.py (hunk 2 above)
Comment (revirtux) [reply index/views.py:47]: הוספתי תיקון גם לCSS

Comment (AmitTPB) [inline index/views.py:47]: image
now the images won't work

**Review by AmitTPB — CHANGES_REQUESTED** [review]

File: index/views.py (hunk 2 above)
Comment (AmitTPB) [reply index/views.py:47]: nvm

**Review by AmitTPB — APPROVED** [review]
Yes it works you can do the thingy now

**Pull request merged**